### ISSUE TITLE
The test results of Panoptic-FPN model based on COCO dataset trained do not match with those of paper named Panoptic Feature Pyramid Networks

### ISSUE THREAD

**Issue by tensorflowt** [facebookresearch/detectron2#604]
hi,I trained the panoramic segmentation model panoptic-fpn based on COCO dataset based on detectron2 framework.My experimental results and the experimental results of the paper are as follows：

Evaluation results for coco_2017_val_panoptic_separated in csv format:
|       method       |   PQ   |   SQ   |   RQ   |   PQ_th   |   SQ_th   |   RQ_th   |  PQ_st   |   SQ_st   |   RQ_st   |
|PanopticFPN_official|40.9    |78.5    |50.1    |48.3       |81.7       |58.3       |29.7      |73.7       |37.3       |
|PanopticFPN_me|30.3 |71.97|37.63|36.087 |77.4451    |44.0588    |21.5678   |63.7070    |27.9257    |

My training parameters are exactly the same as those provided by detectron2, and my training resources are 1 nvidia 2080ti .

What I want to know is how do I need to improve the results of my training model？

thanks

**Comment by ppwwyyxx** [comment]
https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md
As the model zoo said, "Running tools/train_net.py with this config file and 8 GPUs will reproduce the model.".


If you still have questions, please include details about the problem following the [issue template](https://github.com/facebookresearch/detectron2/issues/new?template=unexpected-problems-bugs.md).

**Comment by dragen1860** [comment]
@tensorflowt Hi, I found you are training panoptic segmentation on coco dataset. I am also working on this but I donot know how to prepare the data and training. Can you kindly give me some tips?

**Comment by tensorflowt** [comment]
> @tensorflowt Hi, I found you are training panoptic segmentation on coco dataset. I am also working on this but I donot know how to prepare the data and training. Can you kindly give me some tips?

Please read the file GETTING_STARTED. Md carefully.

**Comment by PkuRainBow** [comment]
@ppwwyyxx Sorry to bother you. 

We have successfully reproduced the results of Panoptic-FPN (ResNet-101) 3x results on the COCO val set (PQ=43.0+) and we directly test the same checkpoint on the COCO test set. But our result on the COCO test is only PQ=33.5 instead of PQ=40.9 as reported in the paper.

We are wondering whether you apply the same checkpoint (trained only on the train set with the default **panoptic_fpn_R_101_3x.yaml**) to generate the results on the COCO test set.

Thanks for your help!

**Comment by ppwwyyxx** [comment]
@PkuRainBow we have not used the code for submission on COCO test-dev. Perhaps the format it produces requires some further post-processing for submission.

**Comment by PkuRainBow** [comment]
@ppwwyyxx Thanks for your reply and we have found the problem. We achieve PQ=43.5 on the COCO test-dev based on the official Panoptic-FPN (ResNet-101) 3x.

**Comment by youngwanLEE** [comment]
@PkuRainBow 

Could you let me know how to get the test-dev results?

When getting the result on test-dev from the evaluation server, it needs both PNGs and JSON files.
